cafelatte/cafelatte.py

```diff
@@ -0,0 +1,24 @@
+from coffee_time.coffee import Coffee
+import math
```

Comment:
Imports should be grouped in the following order:
- standard library imports
- related third party imports
- local application/library specific imports
https://www.python.org/dev/peps/pep-0008/#imports